src/main/scala/org/allenai/plugins/archetypes/WebappPlugin.scala

```diff
@@ -23,8 +23,10 @@ object WebappPlugin extends AutoPlugin {
   override lazy val projectSettings: Seq[Setting[_]] = Seq(
     // Expect the node project in a "webapp" subdirectory.
     NodeKeys.nodeProjectDir in Npm := (baseDirectory in thisProject).value / "webapp",
-    // Force npm:build when using sbt-revolver re-start to ensure UI is built
-    Revolver.reStart <<= Revolver.reStart.dependsOn(NodeKeys.build in Npm),
+    // Run "npm watch" when we run a re-start.
+    Revolver.reStart <<= Revolver.reStart.dependsOn(NodeKeys.nwatch in Npm),
```

Comment:
`re-start` has stopping baked into the task - it doesn't call out to `re-stop`: https://github.com/spray/sbt-revolver/blob/master/src/main/scala/spray/revolver/Actions.scala#L26-L30

As discussed offline, `nwatch` *does* get called each time, but it's smart enough not to spin up extra external processes.